Giraf.IntegrationTests/Endpoints/CitizensEndpointTests.cs

```diff
@@ -236,6 +285,12 @@ public async Task RemoveCitizen_ReturnsNoContent_WhenCitizenExistsInOrganization
                 Assert.NotNull(citizen);
                 var citizenId = citizen.Id;
 
+                var testOrgId = 1;
```

Comment:
Same comment. You're extracting the actual organization in this test, and then just deciding to use id "1" for the claim regardless.